guides/doc-Upgrading_and_Updating/topics/upgrading_clients.adoc

```diff
@@ -68,6 +70,16 @@ For more information, see {ManagingHostsDocURL}configuring-and-setting-up-remote
 # subscription-manager repos \
 --enable={project-client-RHEL7-url}
 ----
+endif::[]
+
+ifdef::katello[]
+. Enable the {project-client-name} repository for this version of {Project}.
++
+[options="nowrap" subs="attributes"]
+----
+# yum install {project-client-RHEL7-url}
```

Comment:
**NOT PART OF YOUR PR**: IMHO this is something we should tackle in the future: the guide is basically made for EL7, but not EL8, SLES, or Debian/Ubuntu.

Reply:
Oh absolutely, still a massive concern I have. Also why I think the new docs still need a huge amount of work. It may be a good thing to focus the Red Hat docs writers on since for Satellite 7.0 it will be needed to have RHEL 8 support.